### ISSUE TITLE
False positive with link discernible text rule

### ISSUE THREAD

**Issue by testingtycoon** [dequelabs/axe-core#613]
I have analyzed a page "https://www.firstrepublic.com/about-us/testimonials/ed-and-peggy-kavounas" and observed that there are 3 violations raised against the link discernible text rule. Third image has a proper alt text, but still axe throws violation against it.

**axe-core version:** 2.4.1
**Code snippet of the element:**
```
<a href="/about-us/testimonials/nancy-and-jeffery-smith" tabindex="-1">
    <figure>
            <!--?xml version="1.0" encoding="iso-8859-1"?-->
			<img src="/~/media/frb/images/testimonial-images/553x350/2015_smith_553x350.ashx?h=166&amp;
			la=en&amp;w=262&amp;hash=4EF45AD92FAC024CDD5B5730B3E05B4374B2DB57" 
			alt="Nancy and Jeffery Smith" title="" width="262" height="166">
    </figure>
</a>
```

**Comment by WilcoFiers** [comment]
The accessible name calculation explicitly ignores the subtree of <figure> elements, only taking in the <figcaption> element if it's there. I think that was a mistake, but I'm not sure, it's built in quite intentional. There might be some AT out there that treats <figure> as decorative if it doesn't have a <figcaption>. Can you test in JAWS and NVDA? I've tested VO and it doesn't seem to care.

**Comment by marcysutton** [comment]
That is interesting. I had no clue what it could be, since there didn't look to be any relevant CSS. Interestingly enough, the computed description for that link showed "Nancy and Jeffery Smith" in chrome://accessibility but the Accessibility Inspection in devtools showed no computed text.

**Comment by WilcoFiers** [comment]
@alice Any thoughts on that one? That sounds like a bug, do you want a report?

**Comment by jnurthen** [comment]
the accessible name calculated by FF for that link is empty so, based on FF, aXe is doing the right thing.

**Comment by WilcoFiers** [comment]
Alright, after some testing, it's not very consistent. NVDA (FF) and VoiceOver (Chrome) read the image, but JAWS (IE & FF) is ignoring it. So even though it's not a problem everywhere, this is a problem. Conclusion, not a false positive.
